onboarder/installation/index.rst

```diff
@@ -0,0 +1,166 @@
+| ONBOARDER_QUEUE_NAME                 | Pub/Sub queue name.                                                                                                               |
++--------------------------------------+-----------------------------------------------------------------------------------------------------------------------------------+
+| ONBOARDER_RETAILER_URL               | URL to the Onboarder retailer. Example : https://retailer-onboarder.akeneo.com                                                    |
++--------------------------------------+-----------------------------------------------------------------------------------------------------------------------------------+
+| ONBOARDER_CLOUD_STORAGE_BUCKET_NAME  | Identifier of the bucket used to share files between your PIM and the retailer Onboarder.                                         |
++--------------------------------------+-----------------------------------------------------------------------------------------------------------------------------------+
+| ONBOARDER_SSO_IDP_PUBLIC_KEY         | Content of the public key (pimmaster.crt)                                                                                         |
++--------------------------------------+-----------------------------------------------------------------------------------------------------------------------------------+
+
+|
+
+**Variables that are specific to your installation**
+
++--------------------------+-------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------+
+| SIMPLESAMLPHP_CONFIG_DIR | Absolute filesystem path to the SSO Identity Provider config directory located in the bundle. Example: ``/var/www/pim/vendor/akeneo/pim-onboarder/src/Infrastructure/Security/SimpleSamlPhp/Configuration`` |
++--------------------------+-------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------+
+| ONBOARDER_PIMMASTER_URL  | Public URL of your Akeneo PIM instance                                                                                                                                                                      |
++--------------------------+-------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------+
+| ONBOARDER_SSO_CERTS_PATH | Absolute path to the SSO certificates that the cloud team provided you in the parameters package. Example: ``/var/www/sso-certs``                                                                           |
```

Comment:
Is it fair to talk about the "cloud team" for customers ? (just so I know ^^)

Reply:
no you're right, I will talk about "Akeneo team"